includes/razorpay-webhook.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
+
+require_once __DIR__.'/../razorpay-sdk/Razorpay.php';
+use Razorpay\Api\Api;
+
+class RZP_Webhook 
+{
+    protected $razorpay; 
+
+    protected $keyId;
+    protected $keySecret;
```

Comment:
this looks duplication. What about creating a WC_Razorpay_Webhook instance in our primary class and using that?